docs/cn/RocketMQ_Example.md

```diff
@@ -440,6 +440,8 @@ public class ScheduledMessageConsumer {
    public static void main(String[] args) throws Exception {
       // 实例化消费者
       DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("ExampleConsumer");
+      // 设置NameServer的地址
+      consumer.setNamesrvAddr("localhost:9876");
```

Comment:
It is better to annotate in English

Reply:
> It is better to annotate in English

agree

Reply:
but it is a documention wrote in Chinese.